src/main/kotlin/no/nav/bidrag/behandling/felles/dto/stonad/HentStonadResponse.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.bidrag.behandling.felles.dto.stonad
+
+import io.swagger.v3.oas.annotations.media.Schema
+import no.nav.bidrag.behandling.felles.enums.StonadType
+import java.time.LocalDateTime
+
+data class HentStonadResponse(
```

Comment:
Samme som for `HentPeriodeResponse`. Tenker at det gir mer mening å kalle klassen `StonadDto`. Tror ikke det er nødvendig å ha med `Hent` og `Response`. Dersom vi ønsker å fremheve hvordan DTO blir brukt, tror jeg det er lurt å definere grensesnittet som et interface som klienter kan implementere: 

```
interface StonadService {
    StonadDto hentStonad();
}
```